travis/install-start-minikube.sh

```diff
@@ -24,7 +24,7 @@ mkdir $HOME/.kube || true
 touch $HOME/.kube/config
 
 export KUBECONFIG=$HOME/.kube/config
-sudo -E ./minikube start --vm-driver=none --use-vendored-driver
```

Comment:
They removed `--use-vendored-driver` in the last version which was released yesterday. 

We can use a specific minikube version to avoid failures like this. (I prefer to use latest for now).